README.md

```diff
@@ -57,7 +57,7 @@ You can ask questions by raising an [issue](https://github.com/EddieHubCommunity
 ### Option 2. Complete this process on your computer (locally)
 
 - Fork the project:
- Fork the project: Click the gray Fork button at the top right of this page. This creates your copy of the project and saves it as a new repository in your GitHub account
+  Click the gray Fork button at the top right of this page. This creates your copy of the project and saves it as a new repository in your GitHub account
```

Comment:
To make it easier, we can add a direct link to fork the repository. What do you think?

[Link here](https://github.com/EddieHubCommunity/hacktoberfest-practice/fork).

Reply:
I agree that, it would be useful if we add the link. However, it contradicts the main motive of this practice. which is exploring and learning things by practicing them yourself first. If we have to add the link then, we can make the whole sentence more simple by just saying, 
Follow the procedure 1 in option 1 or click [here](https://github.com/EddieHubCommunity/hacktoberfest-practice/fork) to [fork the repository](https://github.com/EddieHubCommunity/hacktoberfest-practice/fork). 

that way, learners will read the documentation carefully, and if they face any issue then they can opt for a more easy approach. What do you think?